### PR TITLE
Ignore cgm-with-subnetworks param in cgmes import

### PR DESCRIPTION
**Pull request by basseche**

### CODE REVIEW

File: src/components/dialogs/create-study-dialog/importParametersSection.tsx [hunk 1]
```diff
@@ -59,6 +59,7 @@ const ImportParametersSection: FunctionComponent = () => {
                         onChange={handleParamsChange}
                         variant="standard"
                         selectionWithDialog={(param: any) => param.possibleValues?.length > 10}
+                        disabledParams={['iidm.import.cgmes.cgm-with-subnetworks']} //disabled parameters list
```
Comment (thangqp) [inline src/components/dialogs/create-study-dialog/importParametersSection.tsx]: In my opinion, it is better to extract to an constant outside of component and place in the same file, do not hard-code directly in components.

Comment (basseche) [reply src/components/dialogs/create-study-dialog/importParametersSection.tsx:62]: ok

Comment (basseche) [reply src/components/dialogs/create-study-dialog/importParametersSection.tsx]: solved

**Review by etiennehomer — APPROVED** [review]

**Review by thangqp — APPROVED** [review]
Test OK, code OK

File: src/components/dialogs/create-study-dialog/importParametersSection.tsx [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
-import { FlatParameters, FieldConstants } from '@gridsuite/commons-ui';
-import React, { useState, FunctionComponent } from 'react';
+import { Box, Divider } from '@mui/material';
+import { FieldConstants, FlatParameters, Parameter } from '@gridsuite/commons-ui';
+import { FunctionComponent, useMemo, useState } from 'react';
 import AdvancedParameterButton from './advancedParameterButton';
 import { useController, useWatch } from 'react-hook-form';
-import Box from '@mui/material/Box';
 
+const IGNORED_PARAMS = ['iidm.import.cgmes.cgm-with-subnetworks'];
```
Comment (etiennehomer) [inline src/components/dialogs/create-study-dialog/importParametersSection.tsx:15]: Add a FIXME: To be removed when gridsuite supports subnetworks.

File: src/components/dialogs/create-study-dialog/importParametersSection.tsx [hunk 3]
```diff
@@ -5,13 +5,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { Divider } from '@mui/material';
-import { FlatParameters, FieldConstants } from '@gridsuite/commons-ui';
-import React, { useState, FunctionComponent } from 'react';
+import { Box, Divider } from '@mui/material';
```
Comment (etiennehomer) [inline src/components/dialogs/create-study-dialog/importParametersSection.tsx:8]: Do not change imports ?

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**